File: LanguAI.Frontend/src/api/models/learning-view-model.ts
/* tslint:disable */
/* eslint-disable */
import { LanguageLevelEnum } from '../models/language-level-enum';
export interface LearningViewModel {
  id?: number;
  isActive?: boolean;
  languageLevel?: LanguageLevelEnum;
  learningLanguageCode?: string | null;
  learningLanguageId?: number;
  learningLanguageName?: string | null;
  learningLanguageNameInHun?: string | null;
  nativeLanguageCode?: string | null;
  nativeLanguageId?: number;
  nativeLanguageName?: string | null;
  nativeLanguageNameInHun?: string | null;
  userId?: number;
}

File: LanguAI.Backend/LanguAI.Backend/ViewModels/Learning/LearningViewModel.cs
using LanguAI.Backend.Core.Enums;

namespace LanguAI.Backend.ViewModels.Learning;

public class LearningViewModel
{
    public int Id { get; set; }

    public LanguageLevelEnum LanguageLevel { get; set; }

    public bool IsActive { get; set; }

    public int UserId { get; set; }

    public int LearningLanguageId { get; set; }

    public string LearningLanguageName { get; set; }

    public string LearningLanguageNameInHun { get; set; }

    public string LearningLanguageCode { get; set; }

    public int NativeLanguageId { get; set; }

    public string NativeLanguageName { get; set; }

    public string NativeLanguageNameInHun { get; set; }

    public string NativeLanguageCode { get; set; }
}
